website/src/pages/community.tsx

```diff
@@ -0,0 +1,315 @@
+    if (talk.video_url) {
+      return (
+        <CardActions disableSpacing>
+          <Button size="small" href={talk.conf_url}>Learn more</Button>
+          <Button 
+            size="small" 
+            href={talk.video_url} 
+            sx={{ marginLeft: 2 }}
+          >
+            Watch
+          </Button>
+        </CardActions>
+      )
+    } else {
+      return (
+        <CardActions disableSpacing>
+          <Button size="small" href={talk.conf_url}>Learn more</Button>
+        </CardActions>
+      )
+    }
+  };
+
+  let imgPath = "/img/"+talk.image
+  return (
+    <Card raised={true} sx={{ width: 640 }}>
+    <CardActions disableSpacing sx={{ padding: 0 }}>
+        <ExpandMore
+          expand={expanded}
+          orientation="horizontal"
+          onClick={handleExpandClick}
+          aria-expanded={expanded}
+          aria-label="show more"
+          sx={{ padding: 0 }}
+        >
+          <CardActionArea sx={{ padding: 0 }}>
+            <CardMedia
+              sx={{ width: 640, height: 'auto', padding: 0 }}
+              component="img"
+              src={imgPath}
+              title={talk.conf}
+            />
+          </CardActionArea>
+        </ExpandMore>  
+      </CardActions>
+      <Collapse in={expanded} orientation="vertical" timeout="auto" unmountOnExit> 
+        <CardContent sx={{ width: 640 }}>
+          <Typography variant="h4" color="text.primary" className="pb-5">
+        		{talk.conf}
+        	</Typography>
+        	<Typography variant="h5" color="text.secondary">
+        		{talk.title}
+        	</Typography>
+        	{talk.speakers.map(speaker => 
+	        	<Typography variant="h6" color="text.secondary" key={speaker.toString()}>
+	        		{speaker}
+	        	</Typography>
+        	)}
+          <Typography variant="body1" color="text.secondary" className="pt-5">
+            {talk.description}
+          </Typography>
+        </CardContent>
+        {DisplayLinks()}
+      </Collapse>
+    </Card>
+  )
+}
+
+const FillTalksGrid = ( events: Talk[] ) => {
+  return (
+    <Grid container 
+      margin="auto"
+      rowSpacing={8}
+      columnSpacing={4}
+      paddingBottom={4}
+      width="85%"
+    >
+      {events.map(event => 
+        <Grid
+          marginX="auto"
+          justifyItems="top"
+          key={event.toString()}
+        >
+       		{TalkCard(event)}
+        </Grid >
+      )}
+    </Grid>
+  )
+}
+
+const FillMeetupsGrid = ( events: Meetup[] ) => {
+  return (
+    <Grid container 
+      margin="auto"
+      rowSpacing={8}
+      columnSpacing={4}
+      paddingBottom={4}
+      width="85%"
+    >
+      {events.map(event => 
+        <Grid
+          marginX="auto"
+          justifyItems="top"
+          key={event.toString()}
+        > 
+          {MeetupCard(event)}
+        </Grid >
+      )}
+    </Grid>
+  )
+}
+
+export default function DisplayTalks(): JSX.Element {
+  const seoTitle = 'Talks';
+  const seoDescription = '';
```

Comment:
What's the purpose of this?

Reply:
SEO optimization, thanks for catching this. I added a description.